integrated_channels/utils.py

```diff
@@ -235,6 +235,22 @@ def get_duration_from_estimated_hours(estimated_hours):
     return None
 
 
+def get_courserun_duration_in_days(course_run):
+    """
+    Return the duration in number of days corresponding to a course run.
```

Comment:
Nit: Maybe more accurate: "Return the approximate number of 8-hour days required to complete this course run."